Mathlib/Data/List/Basic.lean

```diff
@@ -1268,17 +1270,18 @@ theorem disjoint_pmap {p : α → Prop} {f : ∀ a : α, p a → β} {s t : List
   rintro b ⟨a, ha, rfl⟩ ⟨a', ha', ha''⟩
   apply h ha
   rwa [hf a a' (hs a ha) (ht a' ha') ha''.symm]
+
 /-- The images of disjoint lists under an injective map are disjoint -/
 theorem disjoint_map {f : α → β} {s t : List α} (hf : Function.Injective f)
     (h : Disjoint s t) : Disjoint (s.map f) (t.map f) := by
-  rw [← pmap_eq_map _ _ _ (fun _ _ ↦ trivial), ← pmap_eq_map _ _ _ (fun _ _ ↦ trivial)]
+  rw [← pmap_eq_map (fun _ _ ↦ trivial), ← pmap_eq_map (fun _ _ ↦ trivial)]
```

Comment:
I'm not sure here, because `H` can usually be found by unification with the LHS, but you are rewriting backwards